plugins/example_collector/test/test_args.py

```diff
@@ -6,4 +6,6 @@ def test_args():
     arg_parser = get_arg_parser()
     ExampleCollectorPlugin.add_args(arg_parser)
     arg_parser.parse_args()
-    assert ArgumentParser.args.example_region is None
+
+
+#    assert ArgumentParser.args.example_arg is None
```

Comment:
left over?

Reply:
Nope, same as with the Config, this is an example of how you would access args had they been added to the Argparser. But I commented the code that adds them so that normal users don't have `--example-arg` listed as one of the worker args.